app/controllers/booths_controller.rb

```diff
@@ -38,6 +56,23 @@ def update
     @url = conference_booth_path(@conference.short_title, @booth.id)
     @booth.update_attributes(booth_params)
 
+    if booth_params[:invited_users]
```

Comment:
There is too much going on here.
The same code is duplicated in 2 different places, and you are using `User.find_by(email: email)` multiple times (it should at least be a variable so that it is easier to read). 

Let's think on how can we make this more clean.

Reply:
@differentreality I will work on improving it.